libs/libglibc-compatibility/glibc-compatibility.c

```diff
@@ -26,7 +26,7 @@ extern "C" {
 
 size_t __pthread_get_minstack(const pthread_attr_t * attr)
 {
-    return 1048576;        /// This is a guess. Don't sure it is correct.
+    return 16384;        /// This is a guess. Don't sure it is correct.
```

Comment:
Please update comment and add explanation.

Reply:
> Please update comment and add explanation.

done